crates/cargo-test-support/src/paths.rs

```diff
@@ -155,6 +159,15 @@ impl CargoPathExt for Path {
             .unwrap_or_else(|e| panic!("failed to mkdir_p {}: {}", self.display(), e))
     }
 
+    fn ls_r(&self) -> Vec<PathBuf> {
+        let mut file_list: Vec<_> = walkdir::WalkDir::new(self)
+            .into_iter()
+            .filter_map(|e| e.map(|e| e.path().to_owned()).ok())
+            .collect();
+        file_list.sort();
```

Comment:
There a reason to manually sort rather than use the sorting built-in to `WalkDir`?

Reply:
Oh, I just didn't know that existed, thanks!